app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,16 @@
+class LikesController < ApplicationController
+  before_action :require_user
+
+  def create
+    @like = Like.new(recipe_id: params[:recipe_id])
```

Comment:
can also add the chef on this line

Matter of fact, you can condense to one line with `Like.create!`

Reply:
I added a flash message if the like is not saved, I think is a good idea keep Like.new(...)